src/Raven.Server/Web/Studio/StudioCollectionsHandlerProcessorForPreviewRevisions.cs

```diff
@@ -40,48 +62,60 @@ protected override IDisposable OpenReadTransaction(DocumentsOperationContext con
     protected override Task WriteItemsAsync(DocumentsOperationContext context, AsyncBlittableJsonTextWriter writer)
     {
         writer.WriteStartArray();
+        if (_totalResults != 0)
```

Comment:
add a comment here that you use `_totalResults` for 3 cases
0 - no results
-1 - unknown number of results
else - we have results

Reply:
Done